From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Xymb <xymb@endcrystal.me>
Date: Tue, 30 Aug 2022 18:26:35 +0200
Subject: [PATCH] Linear region file format.


diff --git a/build.gradle.kts b/build.gradle.kts
index cc5621076421d029aab88d14febc1d3757ead0eb..fff37b8bc6254ce1997634d2b483a13636802568 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -14,6 +14,8 @@ dependencies {
     implementation("io.papermc.paper:paper-mojangapi:1.19.2-R0.1-SNAPSHOT") {
         exclude("io.papermc.paper", "paper-api")
     }
+    implementation("com.github.luben:zstd-jni:1.4.4-2")
+    implementation("org.lz4:lz4-java:1.8.0")
     // LinearPurpur end
     // Paper start
     implementation("org.jline:jline-terminal-jansi:3.21.0")
